src/gl/camera-shader.cpp

```diff
@@ -164,6 +166,7 @@ namespace librealsense
                 if (starts_with(dev_name, "Intel RealSense D415")) index = 0;
                 if (starts_with(dev_name, "Intel RealSense D435")) index = 1;
                 if (starts_with(dev_name, "Intel RealSense L5")) index = 2;
+                if (starts_with(dev_name, "Intel RealSense D455") || starts_with(dev_name, "Intel RealSense D457")) index = 3;
```

Comment:
Let's shorten this to :if (starts_with(dev_name, "Intel RealSense D45"))
It will catch both.

And please verify with both D455 & D457

Reply:
Tested